src/include/pbs_license.h

```diff
@@ -90,33 +89,31 @@ typedef enum node_topology_type ntt_t;
 	  pbs_licensing_license_location : "null" )
 
 enum licensing_backend {
-	LIC_SERVER,	/* reachable license server (to license CPUs) */
 	LIC_SOCKETS,	/* nonzero number of sockets (to license nodes) */
 	LIC_NODES,	/* nonzero number of nodes (to license nodes) */
-	LIC_TRIAL,	/* nonzero number of trial license (to license CPUs) */
 	LIC_UNKNOWN  /* used to hold the value of previous lb */
 };
-
+struct	pbs_lic_counts {
+	long socket_lic_needed;
+	long node_lic_needed;
+	long nsockets;
+	long node_count;
+};
```

Comment:
As discussed, we only need two members in this structure. nsockets and nnodes. This structure can be also used within node structure instead of creating two different variables.

Reply:
This should be declared in pbs_nodes.h and used here. Also can you make the names more generic. like, nsockets and nnodes inside struct devices/inventory?